config/websockets.php

```diff
@@ -4,6 +4,13 @@
 
 return [
 
+    /*
+     * Set a custom dashboard configuration
+     */
+    'dashboard' => [
+        'port' => env('DASHBOARD_PORT', 6001),
```

Comment:
LARAVEL_WEBSOCKETS_PORT would be better it reflects the same naming as the SSL keys at the bottom (in the current websockets.php file at master)

Reply:
Have changed it to LARAVEL_WEBSOCKETS_PORT